packages/api/endpoints/async-operations.js

```diff
@@ -106,11 +118,74 @@ async function del(req, res) {
   return res.send({ message: 'Record deleted' });
 }
 
+/**
+ * Creates a new async operation
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} the promise of express response object
+ */
+async function post(req, res) {
+  const {
+    asyncOperationModel = new AsyncOperationModel({
+      stackName: process.env.stackName,
+      systemBucket: process.env.system_bucket,
+      tableName: process.env.AsyncOperationsTable,
+    }),
+    asyncOperationPgModel = new AsyncOperationPgModel(),
+    knex = await getKnexClient(),
+    esClient = await Search.es(),
+  } = req.testContext || {};
+
+  const apiAsyncOperation = req.body;
+
+  apiAsyncOperation.createdAt = Date.now();
+  apiAsyncOperation.updatedAt = Date.now();
+
+  try {
+    if (!apiAsyncOperation.id) {
+      throw new ValidationError('Async Operations require an ID');
+    }
+    if (await asyncOperationModel.exists({ id: apiAsyncOperation.id })) {
```

Comment:
I think this should be checking whether the async operation exists in PG, not Dynamo, since we're making PG the source of truth in phase 2

Reply:
Oh, good point. I will update this

Reply:
[`d2c9be3` (#2294)](https://github.com/nasa/cumulus/pull/2294/commits/d2c9be37d0bae4433225d328b530cd9be57da96d)